### ISSUE TITLE
Customization

### ISSUE THREAD

**Issue by macsj200** [trufflesuite/drizzle-react-components#69]
I'm quite impressed with how easy it is to use this package to read and write values from a smart contract. That being said, the default view that the component provides isn't what I'm looking for.

Is there any way to customize the output of `<ContractData>`?

**Comment by honestbonsai** [comment]
@macsj200 Can you provide an example of what you're looking for? Or an ideal API for this customization that you're looking for?

**Comment by macsj200** [comment]
Hi, thanks for the quick reply. I actually made a [pull request](https://github.com/trufflesuite/drizzle-react-components/pull/70) for allowing the feature I'm asking for.

Basically, I'm wondering if instead of having `<ContractData>` do the rendering itself (spit out `<li>`'s for example), what if we could pass in another component and use that custom component to render the actual data?

I tried to test my PR, but I was running into build issues using local copies of npm packages. Specifically, I was trying to `yarn link` the package, but my build was complaining that it could no longer find `drizzle-react-components` even though the symlink was in `node_modules/`.

tagging @wesleyfsmith
